src/adapters/InsertVariableHelper.ts

```diff
@@ -0,0 +1,35 @@
+import * as semver from 'semver';
+import { KeyType } from './DebugProtocolAdapter';
+import type { DebugProtocolAdapter, EvaluateContainer } from './DebugProtocolAdapter';
+
+export function insertCustomVariablesHelpers(adapter: DebugProtocolAdapter, expression: string, container: EvaluateContainer) {
+    if (!semver.satisfies(adapter?.activeProtocolVersion, '<3.3.0')) {
```

Comment:
Rather than exiting early, can we wrap the stuff in a `if does satisfy, then do it` block instead. That way it's less of a chance when we add the next batch of changes.

Reply:
Should be addressed now.